{{ cookiecutter.project_slug }}/src/app/settings.py

```diff
@@ -59,6 +59,9 @@
     "corsheaders.middleware.CorsMiddleware",
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
+    {%- if cookiecutter.enable_health_check == "y" %}
+    "helpers.health_check_middleware.HealthCheckMiddleware",
+    {%- endif %}
```

Comment:
I believe the linter fails over here because `{%- endif %}` will probably expand into a new line. You can solve this by using the following:

```suggestion
    {%- if cookiecutter.enable_health_check == "y" %}
    "helpers.health_check_middleware.HealthCheckMiddleware",{%- endif %}
```